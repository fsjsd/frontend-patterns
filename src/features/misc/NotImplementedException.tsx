import React from 'react'
import ContentHeader, { ContentHeaderLabel } from '../../ux/ContentHeader'
import { ContentWrapper } from '../../ux/ContentWrapper'
import Loading from '../../ux/Loading'

const NotImplementedException = () => {
  return (
    <>
      <ContentHeader>
        <ContentHeaderLabel>throw new NotImplementedException()</ContentHeaderLabel>
      </ContentHeader>
      <ContentWrapper>
        <p>Coming Soon!</p>
        <Loading />
      </ContentWrapper>
    </>
  )
}

export default NotImplementedException