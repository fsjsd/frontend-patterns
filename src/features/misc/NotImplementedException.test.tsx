import React from 'react'
import { render } from '@testing-library/react'
import NotImplementedException from './NotImplementedException'

describe('NotImplementedException', () => {
  test('render matches snapshot', () => {
    const { container } = render(<NotImplementedException />)
    expect(container).toMatchSnapshot()
  })
})
