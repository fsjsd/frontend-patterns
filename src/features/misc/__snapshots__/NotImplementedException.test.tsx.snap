// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotImplementedException render matches snapshot 1`] = `
<div>
  <div
    class="sc-bczRLJ dtMXGe"
  >
    <div
      class="sc-gsnTZi dkPlsI"
    >
      throw new NotImplementedException()
    </div>
  </div>
  <div
    class="sc-dkzDqf eGEKCX"
  >
    <p>
      Coming Soon!
    </p>
    <div
      class="sc-hKMtZM hbOElo"
    >
      <div />
      <div />
    </div>
  </div>
</div>
`;
